WordPress/src/main/java/org/wordpress/android/modules/AppComponent.java

```diff
@@ -254,6 +255,7 @@
 
     void inject(HtmlToSpannedConverter object);
 
+    void inject(PluginBrowserActivity object);
     void inject(PluginDetailActivity object);
     void inject(PluginBrowserViewModel object);
```

Comment:
I don't think we need this anymore now that the injection is done with DI. (Quick reminder: We'll need to remove the import for it as well.)

Reply:
Fixed